b1ddi/resource_dns_record_test.go

```diff
@@ -201,6 +203,21 @@ func TestAccResourceDnsRecord_Update(t *testing.T) {
 						ttl = 24400
 						type = "A"
 						zone = b1ddi_dns_auth_zone.tf_acc_test_auth_zone.id
+					}
+					resource "b1ddi_dns_record" "tf_acc_test_dns_srv_record" {
+						comment = "This DNS Record is created by the terraform provider acceptance test"
+						disabled = true
+						name_in_zone = "tf_acc_test_srv_record"
+						rdata = {
```

Comment:
Do we get 0 for `priority`?

Reply:
Do you mean in the API response?
In the updater function, we won't be setting it to options if the key is not present.

Reply:
The RDATA for SRV has `priority` field along with the two you have in unit-test. Just asking as I would expect people to fill-in `priority` as well.

Reply:
It's a mandatory field. Will update the UT

Reply:
Do you need to make the change here as well?

Reply:
This shouldn't be here as we have separate UT for SRV record. Will remove it.